include/ppx/application.h

```diff
@@ -254,6 +254,10 @@ struct StandardOptions
 
     std::shared_ptr<KnobFlag<std::vector<std::string>>> pAssetsPaths;
     std::shared_ptr<KnobFlag<std::vector<std::string>>> pConfigJsonPaths;
+
+    std::shared_ptr<KnobFlag<bool>> pEnableFDM;
+    std::shared_ptr<KnobFlag<bool>> pEnableVRS;
+    std::shared_ptr<KnobFlag<bool>> pDisableShadingRate;
```

Comment:
Is this flag really needed? Seems redundant given we have the 2 others to control VRS.

Reply:
Yeah, I think it makes sense to have all three. The absence of any of these flags indicates to use whatever default the application specified in `ApplicationSettings`, and `--disable-shading-rate` explicitly says to use the `SHADING_RATE_NONE`.

Reply:
This is superseded by the other conversation about knobs below. `--disable-shading-rate` is now `--shading-rate-mode=none`.